scripts/bx_auth.sh

```diff
@@ -15,7 +15,7 @@ BLUEMIX_SPACE="dev"
 is_pull_request "$0"
 
 echo "Authenticating to Bluemix"
-bx login -a https://api.ng.bluemix.net
+bx login -a https://api.ng.bluemix.net --apikey "$BLUEMIX_API_KEY"
```

Comment:
This is not needed. When the environment variable is set, --apikey is redundant.